src/app/pcasts/dao/series_dao.py

```diff
@@ -1,14 +1,21 @@
+from app.pcasts.dao import episodes_dao
 from . import *
 
 def get_series(series_id, user_id):
   series = Series.query.filter(Series.id == series_id).first()
   series.is_subscribed = is_subscribed_by_user(series_id, user_id)
+  most_recent_episode = Episode.query.filter(Episode.series_id == series_id) \
+    .order_by(Episode.created_at.desc()).limit(1).first()
+  series.last_updated = most_recent_episode.created_at
   return series
 
 def get_multiple_series(series_ids, user_id):
   series = Series.query.filter(Series.id.in_(series_ids)).all()
   for s in series:
     s.is_subscribed = is_subscribed_by_user(s.id, user_id)
+    most_recent_episode = Episode.query.filter(Episode.series_id == s.id) \
```

Comment:
This is pretty expensive - we should probably do a group by combined with a max on the created_at field

Reply:
As a follow-up, [this](https://stackoverflow.com/questions/4510185/select-max-value-of-each-group) is a good post to checkout, except we'd be doing a join + a group by on max, rather than querying from a single table.